tests/integration/test_api.py

```diff
@@ -470,3 +477,45 @@ def test_annotate_insight_anonymous_then_authenticated(client, mocker):
         server_domain=settings.OFF_SERVER_DOMAIN,
         auth=OFFAuthentication(username="a", password="b"),
     )
+
+
+def test_get_images(client):
+    result = client.simulate_get("/api/v1/images?page=1&count=25")
+    assert result.status_code == 200
+
+    image_model_factory1 = ImageModelFactory(barcode="123")
+    image_model_factory2 = ImageModelFactory(barcode="456")
+
+    image_prediction_factory1 = ImagePredictionFactory(image=image_model_factory1.id)
+    image_prediction_factory2 = ImagePredictionFactory(image=image_model_factory2.id)
+    image_prediction_factory3 = ImagePredictionFactory()
```

Comment:
For your test, you'd better want to have an image without ImagePrediction attached to it, to test your with_predicted=False

So here you'd better only create an image.
More over I will intentionally reuse barcode "123" to have two images returned on this barcode with with_predicted=True 

```suggestion
    image_model3 = ImageModelFactory(barcode="123")
```